elastic/kibana
# [Cases] Add MTTR metric

## Summary

This PR adds the MTTR (mean time to response) metric for all cases. It follows the same pattern as with the single case metrics.

Depends on https://github.com/elastic/kibana/pull/130448

## To verify

1. Create a case and leave it open for a while. 
2. Close the case.
3. Do the following request: `GET /api/cases/metrics/?features=["mttr"]`

The endpoint accepts the following query parameters:

- `owner`: string | string[]
- `from`: an ISO date
- `to`: an ISO date

Try different values to verify that the endpoint respects the query parameters.

### Checklist

Delete any items that are not applicable to this PR.



### For maintainers
